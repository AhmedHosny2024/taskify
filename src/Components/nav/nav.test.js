import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import MenuAppBar from './nav';

test('snapshot', async () => {
    
    const tree = renderer.create(
      <MenuAppBar/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });