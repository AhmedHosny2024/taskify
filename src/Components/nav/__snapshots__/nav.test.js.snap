// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="MuiBox-root css-i9gxme"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1fxzt37-MuiToolbar-root"
    >
      <div
        className="MuiTypography-root MuiTypography-h4 css-1qwyl82-MuiTypography-root"
      >
        Taskify
      </div>
      <div
        className="css-u4p24i"
      >
        <div
          className="css-1522hk"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
            data-testid="NotificationsNoneIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.9 2 2 2zm6-6v-5c0-3.07-1.63-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.64 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2zm-2 1H8v-6c0-2.48 1.51-4.5 4-4.5s4 2.02 4 4.5v6z"
            />
          </svg>
        </div>
        <div
          className="css-u4p24i"
        >
          <div
            className="css-1522hk"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
              data-testid="AccountCircleIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 4c1.93 0 3.5 1.57 3.5 3.5S13.93 13 12 13s-3.5-1.57-3.5-3.5S10.07 6 12 6zm0 14c-2.03 0-4.43-.82-6.14-2.88C7.55 15.8 9.68 15 12 15s4.45.8 6.14 2.12C16.43 19.18 14.03 20 12 20z"
              />
            </svg>
          </div>
          <p
            className="css-xdzyr5"
          >
            Hosny
          </p>
        </div>
      </div>
    </div>
  </header>
</div>
`;
